src/utils/errors/constants.ts

```diff
@@ -0,0 +1,92 @@
+import { CustomErrorMapping } from "./types";
+
+export const STRING_PREFIX_AFTER_UNICODE_REPLACEMENT = 'y %';
+
+// eslint-disable-next-line no-useless-escape
+export const INVALID_ASCII_REGEX = /[^A-Za-z 0-9 \.,\?""!@#\$%\^&\*\(\)-_=\+;:<>\/\\\|\}\{\[\]`~]*/g;
```

Comment:
A neat little trick (that begs a comment), but all the printable ascii characters are in the range from ` ` (space) to `~` (tilde), so the regex can be compressed to: `/[^ -~]+/g`. Is this better? I think so (with a comment explaining what it is) since it ensures no printable ascii chars were missed, despite being more obscure knowledge, where a list like the above requires you to check that all symbols were included.